lib/pangea/extensions/pangea_room_extension/room_analytics_extension.dart

```diff
@@ -103,17 +103,19 @@ extension AnalyticsRoomExtension on Room {
         .where((teacher) => !participants.contains(teacher))
         .toList();
 
-    Future.wait(
-      uninvitedTeachers.map(
-        (teacher) => analyticsRoom.invite(teacher.id).catchError((err, s) {
-          ErrorHandler.logError(
-            e: err,
-            m: "Failed to invite teacher ${teacher.id} to analytics room ${analyticsRoom.id}",
-            s: s,
-          );
-        }),
-      ),
-    );
+    if (!canSendEvent(EventTypes.SpaceChild)) {
```

Comment:
Hey @Kelrap did you mean to put the 'not' operator here? It should just be canSendEvent, without the 'not'. Also, in this case the permission for inviting teachers to analytics rooms would be on the level of the analyticsRoom, and for the event type EventTypes.RoomMember.

Can you also add a permission check to the function that adds analytics room to spaces as well? In that case, the event type would be EventTypes.SpaceChild